src/friendly-iframe-embed.js

```diff
@@ -580,6 +626,41 @@ export class FriendlyIframeEmbed {
   }
 }
 
+/**
+ * @param {!Window} win
+ * @param {!Element} ampAdParent
+ * @param {!Element} ampLightbox
+ * @visibleForTesting
+ */
+export function renderCloseButtonHeader(win, ampAdParent, ampLightbox) {
+  const el = htmlFor(ampAdParent)`
+    <i-amphtml-ad-close-header role=button>
+      <div>Ad</div>
+      <i-amphtml-ad-close-button class="amp-ad-close-button">
```

Comment:
needs `role=button` `tabIndex=0` `aria-label="Close ad"`

Reply:
Added to `i-amphtml-ad-close-header`. `i-amphtml-ad-close-button` has no pointer events, so no need to set it on it.